### DISCUSSION TITLE
Problem with stacked generate

### DISCUSSION THREAD

**Discussion by eraldss1** [FilledStacks/stacked#920] [General]
Hey, i'm new at Stacked CLI. I tried to create new app with Stacked CLI and get some errors with build_runner. Can someone explain what happen? 

The error i got:

`Failed to build build_runner:build_runner:
/C:/Users/mess/AppData/Local/Pub/Cache/hosted/pub.dev/build_runner-2.3.3/lib/src/generate/watch_impl.dart:315:10: Error: The method 'debounceBuffer' isn't defined for the class 'Stream<AssetChange>'.
 - 'Stream' is from 'dart:async'.
 - 'AssetChange' is from 'package:build_runner/src/watcher/asset_change.dart' ('/C:/Users/mess/AppData/Local/Pub/Cache/hosted/pub.dev/build_runner-2.3.3/lib/src/watcher/asset_change.dart').
Try correcting the name to the name of an existing method, or defining a method named 'debounceBuffer'.
        .debounceBuffer(_debounceDelay)
         ^^^^^^^^^^^^^^
/C:/Users/mess/AppData/Local/Pub/Cache/hosted/pub.dev/stream_transform-2.1.0/lib/src/rate_limit.dart: Error: The control character U+0000 can only be used in strings and comments.
/C:/Users/mess/AppData/Local/Pub/Cache/hosted/pub.dev/stream_transform-2.1.0/lib/src/async_map.dart:45:12: Error: The method 'buffer' isn't defined for the class 'Stream<T>'.
 - 'Stream' is from 'dart:async'.
Try correcting the name to the name of an existing method, or defining a method named 'buffer'.
    return buffer(workFinished.stream)._asyncMapThen(convert, workFinished.add);
           ^^^^^^
/C:/Users/mess/AppData/Local/Pub/Cache/hosted/pub.dev/build_runner-2.3.3/lib/src/daemon/daemon_builder.dart:249:10: Error: The method 'debounceBuffer' isn't defined for the class 'Stream<WatchEvent>'.
 - 'Stream' is from 'dart:async'.
 - 'WatchEvent' is from 'package:watcher/src/watch_event.dart' ('/C:/Users/mess/AppData/Local/Pub/Cache/hosted/pub.dev/watcher-1.0.2/lib/src/watch_event.dart').
Try correcting the name to the name of an existing method, or defining a method named 'debounceBuffer'.
        .debounceBuffer(buildOptions.debounceDelay);
         ^^^^^^^^^^^^^^
pub finished with exit code 1`

**Comment by FilledStacks** [comment]
Hey, what version of stacked_cli are you using?

**Reply by eraldss1** [reply]
i use stacked version 1.5.2

**Reply by eraldss1** [reply]
do you have any solution?

**Reply by FilledStacks** [reply]
I don't. I haven't seen the problem before. Are you on Windows or Mac?

**Reply by eraldss1** [reply]
i'm on windows 10 64-bit

**Reply by FilledStacks** [reply]
I have no idea what that is. Can you install a different dart cli tool, like very_good_cli and see if you get the same issue
